### PR TITLE
Prepared release of version 1.10.1

### PR DESCRIPTION
**Pull request by tabergma**
**Proposed changes**:
Prepared release of version 1.10.1

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -17,6 +17,30 @@ This project adheres to `Semantic Versioning`_ starting with version 1.0.
...
+- `#5638 <https://github.com/rasahq/rasa/issues/5638>`_: Fix: DIET breaks with empty spaCy model
+- `#5755 <https://github.com/rasahq/rasa/issues/5755>`_: Remove ``clean_up_entities`` from extractors that extract pre-defined entities.
+  Just keep the clean up method for entity extractors that extract custom entities.
+- `#5792 <https://github.com/rasahq/rasa/issues/5792>`_: Fixed issue where the ``DucklingHTTPExtractor`` component would
+  not work if its `url` contained a trailing slash.
+- `#5825 <https://github.com/rasahq/rasa/issues/5825>`_: Fix list index out of range error in ``ensure_consistent_bilou_tagging``.
+
+Miscellaneous internal changes
+------------------------------
+- #5788
```
Comment (akelad) [inline CHANGELOG.rst:41]: what happened here? this should have a description right?

Comment (tabergma) [reply CHANGELOG.rst:41]: no, internal changes does not have descriptions in the changelog, that is intended I guess

Comment (tabergma) [reply CHANGELOG.rst:41]: see previous releases: https://rasa.com/docs/rasa/changelog/#miscellaneous-internal-changes

Comment (akelad) [reply CHANGELOG.rst:41]: ok, why not 🤷

**Pull request merged**